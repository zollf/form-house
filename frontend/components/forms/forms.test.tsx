import React from 'react';
import { render } from '@testing-library/react';

import Forms from './';

it('matches its snapshot', () => {
  expect(render(<Forms />).baseElement).toMatchSnapshot();
});