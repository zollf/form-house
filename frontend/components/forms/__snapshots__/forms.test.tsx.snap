// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches its snapshot 1`] = `
<body>
  <div>
    <h1
      class="forms"
    >
      React - Forms
    </h1>
  </div>
</body>
`;
